markdown/developer_docs.md

```diff
@@ -36,30 +39,31 @@ _(any point can be skipped, given a good enough reason...)_
 
 If possible, it's great if pipelines can also have:
 
-* Software bundled using [bioconda](https://bioconda.github.io/)
+* All software bundled using [bioconda](https://bioconda.github.io/)
     * See below for how to use a bioconda env script automatically build docker and singularity containers, meaning you have a single file to maintain.
 * Optimised output file formats
     * Pipelines should generate `CRAM` alignment files by default, but have a `--bam` option to generate `BAM` outputs if required by the user.
 * Explicit support for running in cloud environments
-    * For example, use of [AWS-iGenomes](https://ewels.github.io/AWS-iGenomes/)
+    * For example, use of [AWS-iGenomes](https://ewels.github.io/AWS-iGenomes/) and aws-batch
 * Benchmarks from running on cloud environments such as [AWS](https://aws.amazon.com/)
 
 ## Pipeline organisation
-It's highly recommended that pipelines are built using the **nf-core tool** [`create`](tools#create) command, as this will ensure future community updates to be propagated to your pipeline (read more [here](sync)).
-
+It's highly recommended that pipelines are built using the [`nf-core create`](tools#create) command, as this will ensure future community updates to be propagated to your pipeline (read more [here](sync)).
```

Comment:
This is a repeat from line 17, might be on purpose?

Reply:
It is a duplication yes. The duplication is not introduced here though, but yes it could use a review. I guess it doesn't hurt to repeat some things...